encrypted-media-respec.html

```diff
@@ -2337,7 +2345,7 @@
             <li><p>Let the <var>session</var> be the associated <a>MediaKeySession</a> object.</p></li>
             <li><p>Let <var>expiration time</var> be <code>NaN</code>.</p></li>
             <li><p>If the new expiration time is not <code>NaN</code>, let <var>expiration time</var> be that expiration time.</p></li>
-            <li><p>Set the <var>session</var>'s <a def-id="expiration"></a> attribute to <var>expiration time</var>.</p></li>
+            <li><p>Set the <var>session</var>'s <a def-id="expiration"></a> attribute to <var>expiration time</var> expressed as <a href="#time">time</a>.</p></li>
```

Comment:
This new text isn't strictly necessary, but it might be nice to reference the time definition again where we expose it to the app. A NOTE might be more appropriate but would probably be distracting.